accounts/client.go

```diff
@@ -24,6 +24,26 @@ func NewClient(
 	}
 }
 
+func (c *Client) SubmitFile(file File) (*common.IdResponse, error) {
```

Comment:
Lets relocate this method to the bottom of this file.

Reply:
And why did we rename this method? UpdateFile was ok I believe

Reply:
I have followed a .net example